web/components/headers/cycle-issues.tsx

```diff
@@ -197,15 +197,24 @@ export const CycleIssuesHeader: React.FC = observer(() => {
                     label={
                       <>
                         <ContrastIcon className="h-3 w-3" />
-                        <div className=" w-auto max-w-[70px] sm:max-w-[200px] inline-block truncate line-clamp-1 overflow-hidden whitespace-nowrap">
-                          {cycleDetails?.name && cycleDetails.name}
+                        <div className="flex items-center gap-2 w-auto max-w-[70px] sm:max-w-[200px] truncate">
+                          <p className="truncate">{cycleDetails?.name && cycleDetails.name}</p>
+                          {cycleDetails && issueFilters?.displayFilters && (
+                            <span className="flex items-center text-center justify-center px-2 flex-shrink-0 bg-custom-primary-100/20 text-custom-primary-100 text-xs font-semibold rounded-xl">
+                              {issueFilters.displayFilters.sub_issue
```

Comment:
@anmolsinghbhatia could you verify the figma files? this seems like it has a toaster on the issue count in the design and Also there seems to be designs for issue counts when filters are applied as well, could you check?